reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -112,14 +112,53 @@ function renderLicensesTable (
   for (let i = 0; i < columnNames.length; i++)
     columnNames[i] = chalk.blueBright(columnNames[i])
 
-  return table(
-    [
-      columnNames,
-      ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
-        .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
-    ],
-    TABLE_OPTIONS
-  )
+  const data = [
+    columnNames,
+    ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
+      .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
+  ]
+  let detailsColumnMaxWidth = 40
+  let packageColumnMaxWidth = 0
+  let licenseColumnMaxWidth = 0
+  if (opts.long) {
+    // Use the package link to determine the width of the details column
+    detailsColumnMaxWidth = licensePackages.reduce((max, pkg) => Math.max(max, pkg.homepage?.length ?? 0), 0)
+    for (let i = 1; i < data.length; i++) {
+      const row = data[i]
+      const texts = row[2].split('\n')
+      const repeatCount = Math.max(0, texts.length - 1)
+      row[0] += '\n '.repeat(repeatCount) // Add extra spaces to the package column
+      row[1] += '\n '.repeat(repeatCount) // Add extra spaces to the license column
```

Comment:
are we sure that these rows will always be only a single line?

Reply:
From what I understand, the name of the package should only be one line. Even if the license has multiple values, they seem to be spliced via OR, leaving only one row.